debian/control

```diff
@@ -54,27 +52,29 @@ Build-Depends: bc,
                python (>= 2.7),
                python-all-dev,
                python-cherrypy3,
-               python-jwt,
-               python-nose,
-               python-pecan,
-               python-bcrypt,
-               python-six,
-               python-tox,
-               python-coverage,
-               python-prettytable,
-               python-setuptools,
+# Make-Check   python-jwt,
+# Make-Check   python-nose,
+# Make-Check   python-pecan,
+# Make-Check   python-bcrypt,
+# Make-Check   python-six,
+# Make-Check   python-tox,
+# Make-Check   python-coverage,
+# QA-Run       python-openssl,
+# Make-Check   python-prettytable,
+# QA-Run       python-requests,
```

Comment:
Python requests is needed in "Ansible" and "DeepSea" Orchestrators modules....  i'm not sure if the label"QA-Run" will prevent to install this library if the environment it it is not QA...

Reply:
See: https://tracker.ceph.com/issues/37584

Reply:
@jmolmo they are already listed as runtime dependencies of `ceph-common`. `QA-Run` helps `install-deps.sh` to identify that they also required for a full-blown Ceph developing environment. please see https://tracker.ceph.com/issues/37584#note-15 for more context.

Reply:
Excellent explanation of the problem in your comment the bug. Thank you for it. LGTM